docs/outputs.md

```diff
@@ -0,0 +1,27 @@
+# Outputs
```

Comment:
Why is this file showing as new. Maybe merge master into your branch to resolve diffs? Presumably this is in master already.